index.js

```diff
@@ -79,10 +79,33 @@ program
       });
   });
 
-program.command("format").action(() => {
+program
+  .command("format")
+  .option("-w, --went <spec>", "Display days at the gym according to spec: char[:color]")
+  .option("-a, --away <spec>", "Display days away from the gym according to spec: char[:color]")
+  .action(options => {
+
   chalk.enabled = 1;
   chalk.level = 3;
 
+  // Character and its background color
+  const defaultWent   = ["\u25ac", 220];
+  const defaultAway = ["\u25ac", 110];
+
+  let wentChar = defaultWent;
+  if(options.went) {
+    wentChar = options.went.split(":", 2);
```

Comment:
Not bulletproof. User can specify `:100` and `wentChar[0]` will be an empty string.